core/src/main/resources/lib/form/textarea.jelly

```diff
@@ -89,6 +89,7 @@ THE SOFTWARE.
             readonly="${attrs.readonly}"
             codemirror-mode="${attrs['codemirror-mode']}"
             codemirror-config="${attrs['codemirror-config']}">
+    <j:if test="${value != null &amp;&amp; !empty(value.toString()) &amp;&amp; (value.toString().codePointAt(0) == 10 || value.toString().codePointAt(0) == 13)}"><j:whitespace>&#10;</j:whitespace></j:if>
```

Comment:
[Documentation for Jelly](http://commons.apache.org/proper/commons-jelly/tags.html#jelly:core) is weak. Generally speaking if you are trying to do anything subtle you have to switch to Groovy to get more control.